src/quest/__init__.py

```diff
@@ -31,3 +33,14 @@ def create_history(wid: str) -> History:
         return PersistentHistory(wid, LocalFileSystemBlobStorage(save_folder / namespace / wid))
 
     return WorkflowManager(namespace, storage, create_history, factory)
+
+def print_directory_json(folder: Path):
+    for file in sorted(folder.iterdir()):
+        if os.path.isdir(file):
+            print_directory_json(file)
+        else:
+            print(f"Current Directory: {folder.name}")
+            content = json.loads(file.read_text())
+            print(json.dumps(content, indent=2))
+            print("\n")
+
```

Comment:
This was a hack for the demo. It is **not** something I want users doing.

Reply:
So I would not include this in `quest`

Reply:
Got it 👍